parse_bench_test.go

```diff
@@ -38,15 +37,15 @@ func getLargeConfig(b *testing.B) (string, func()) {
 	path := getTestConfigPath("large-config", "nginx.conf.bz2")
 
 	// unpack compressed log file and place in a temporary directory
-	f, e := os.Open(path)
-	if e != nil {
+	file, err := os.Open(path)
+	if err != nil {
 		b.Skip("cannot open input file")
 	}
-	defer f.Close()
+	defer file.Close()
 
 	// Open output file
-	tmpdir, e := ioutil.TempDir("", "alog")
-	if e != nil {
+	tmpdir, err := os.MkdirTemp("", "alog")
```

Comment:
inside tests, you can use `t.TmpDir()` without error, or `b.TmpDir()` in this case

Reply:
That's good to know. Thanks for this suggestion!